session.rb

```diff
@@ -0,0 +1,45 @@
+require_relative "track" 
+
+class Session
+  attr_reader :talks, :year, :month, :day
+
+  def initialize(talks)
```

Comment:
A session doesn't need to hold reference of all tracks. The initialization is incorrect.